README.md

```diff
@@ -1,141 +1,54 @@
 # Lifecycle Manager
 
-Kyma is the opinionated set of Kubernetes based modular building blocks that includes the necessary capabilities to develop and run enterprise-grade cloud-native applications. Kyma's Lifecycle Manager (or `lifecycle-manager` in various technical references) is a tool that manages the lifecycle of these components in your cluster.
+Kyma is an opinionated set of Kubernetes-based modular building blocks that includes the necessary capabilities to develop and run enterprise-grade cloud-native applications. Kyma's Lifecycle Manager is a tool that manages the lifecycle of these modules in your cluster.
 
-### Contents
-* [How it works](#how-it-works)
-  * [Example](#example)
-  * [Getting Started](#getting-started)
-* [Architecture](#architecture)
-  * [Stability](#stability)
-* [Deployment / Delivery models](#deployment--delivery-models)
-  * [Release Lifecycles for Modules](#release-lifecycles-for-modules)
-  * [Versioning and Releasing](#versioning-and-releasing)
-  * [Comparison to the Old Reconciler](#comparison-to-the-old-reconciler)
-* [Testing and implementation guide](#testing-and-implementation-guide)
+## Basic Concepts and Modularization
 
-## How it works
+See the list of basic concepts relating to Lifecycle Manager to understand its workflow better.
 
-Lifecycle Manager manages Clusters through the [Kyma Custom Resource](api/v1beta1/kyma_types.go), which contains a desired state of all modules in a cluster. Imagine it as a one stop shop for a cluster where you can add and remove modules with domain-specific functionality with additional configuration.
+- Kyma custom resource (CR) - [short description]
+- ModuleTemplate CR - [short description]
+- Module CR - [short description]
+- Manifest CR - [short description]
```

Comment:
I think we can skip this as it is rather internal thing that the users that do not go deep into the technical side of the KLM will never notice